### ISSUE TITLE
Create Cluster: Cannot navigate to Show Advanced via keyboard

### ISSUE THREAD

**Issue by rak-phillip** [rancher/dashboard#14306]
**Setup**
- Rancher version: v2.12-head
- Rancher UI Extensions: NA
- Browser type & version: Firefox & Chrome latest

**Describe the bug**

It's not possible to navigate to and activate the Show Advanced feature when creating Machine Pools for a cluster.

**To Reproduce**

1. Navigate to Cluster Management and click on Create
2. Click on Amazon EC2
3. Enter a cluster name
4. ⚠️ Using keyboard only, navigate to Show Advanced under Machine Pools

**Result**

It's not possible to focus on Show Advanced

**Expected Result**

Show Advanced can receive keyboard focus and can be activated via keyboard

**Screenshots**

NA

**Additional context**

**Comment by a-arias** [comment]
**TESTS**
Rancher : v2.13-64cd70e41d1937fb72f4982d4e70a0b1f4a13a8d-head
UI:master https://github.com/rancher/dashboard/commit/ef6d0a8ffb06d136d97619ac92f7bfab1fc1829b
Fix verified ✅

https://github.com/user-attachments/assets/515729c4-01ee-4c05-bcd4-bdc9759e39df
